v2/handler.go

```diff
@@ -148,6 +159,6 @@ func (f StreamTransportHandlerFunc) HandleStream(stream *ServerStream) error {
 }
 
 // Handle handles an inbound unary request.
-func (f UnaryEncodingHandlerFunc) Handle(ctx context.Context, r *Request, rb interface{}) (*Response, interface{}, error) {
-	return f(ctx, r, rb)
+func (f UnaryEncodingHandlerFunc) Handle(ctx context.Context, b interface{}) (interface{}, error) {
```

Comment:
nit: what is `b`? Should this be `req` or `r`?

Reply:
`b` was for "body" or "buffer", will use "reqBody" instead.